app/controllers/admin/company_values_controller.rb

```diff
@@ -38,8 +38,12 @@ def update
 
     def destroy
       @company_value = CompanyValue.find(params[:id])
-      @company_value.destroy
-      redirect_to admin_company_values_url, notice: 'Company Value was successfully destroyed.'
+      if @company_value.destroy
+        redirect_to admin_company_values_url, notice: 'Company Value was successfully destroyed.'
+      else
+        redirect_to admin_company_values_url
```

Comment:
you could write it like this:
```
redirect_to admin_company_values_url, notice: @company_value.errors.full_messages[0]
```

If you don't like the shorthand syntax then it's fine, but I'd set the flash message before the redirection.
Btw. it's a great example showing that redirect_to doesn't stop the execution of an action 🙌

Reply:
Thanks for the hint.